Makefile

```diff
@@ -846,3 +846,6 @@ else
 	 go run update_kubernetes_version.go)
 endif
 
+.PHONY: stress
+stress:
+	go test -test.v -test.timeout=2h ./test/stress -loops=10
```

Comment:
lets add output file so we can use in gopogh report like here https://github.com/medyagh/minikube/blob/89a8cdc794098f9ba0550fbb5eeb19bc3dd8774d/Makefile#L304
```
go test -test.v -test.timeout=2h ./test/stress -loops=10 | tee "./out/stresstestout_$(COMMIT_SHORT).txt"
```

also lets add a stress section for html report too

.PHONY: html_report
html_report: ## Generate HTML  report out of the last ran integration test logs.

Reply:
done, I made it so that we can reuse the existing html_report make target. i doubt we'll have to worry too much about local integration and stress tests trampling each other, and this keeps it simple.